core/src/main/resources/hudson/logging/LogRecorderManager/all.jelly

```diff
@@ -31,6 +31,9 @@ THE SOFTWARE.
   <st:include page="sidepanel.jelly" />
   <l:main-panel>
     <h1><l:icon class="icon-clipboard icon-xlg"/>${%Jenkins Log}</h1>
+    <div class="alert alert-info">
+      Log messages at a level lower than INFO are never recorded in the Jenkins log. Use <a href=".">log recorders</a> to record these log messages.
```

Comment:
```suggestion
      Log messages at a level lower than INFO are never recorded in the Jenkins system log. Use <a href=".">log recorders</a> to record these log messages.
```

Reply:
It's a bit more complicated than that.

The breadcrumb you're referring to now matches the label on `/manage`, which was the goal of the change.

The `LogRecorderManager` UI includes both what is called "Jenkins log" here, and "All Log Messages" (previously "All Logs") in the sidepanel, as well as log recorders and the global levels overview.

So if anything should be changed, it's that this doesn't say "Log messages … in all log messages" which I didn't want to write, because it makes no sense at all.

In the interest of limiting the scope of this PR, I wrote a message I hoped could be understood, and moved on.